pkg/controller/migplan/migplan_controller.go

```diff
@@ -523,96 +518,109 @@ func (r ReconcileMigPlan) ensureMigAnalytics(plan *migapi.MigPlan) error {
 	pvMigAnalytics.Spec.MigPlanRef = &kapi.ObjectReference{Namespace: plan.Namespace, Name: plan.Name}
 	err = r.Create(context.TODO(), pvMigAnalytics)
 	if err != nil {
-		return err
+		return liberr.Wrap(err)
 	}
 	return nil
 }
 
-func (r ReconcileMigPlan) waitForMigAnalyticsReady(plan *migapi.MigPlan) (*migapi.MigAnalytic, error) {
+func (r *ReconcileMigPlan) waitForMigAnalyticsReady(plan *migapi.MigPlan) (*migapi.MigAnalytic, error) {
```

Comment:
This method name should reflect that we're _checking_ if MigAnalytics is ready, we aren't waiting and blocking.

Reply:
@djwhatle fixed